### DISCUSSION TITLE
Line up logos in new design?

### DISCUSSION THREAD

**Discussion by anthonyshew** [fuma-nama/fumadocs#452] [General]
The new design is awesome. Shipped it already on turbo.build.

I noticed the logo has a slight drift between the DocsLayout and non-docs layout. 
https://github.com/fuma-nama/fumadocs/assets/35677084/66bcc79f-4caa-4d9a-b86f-8e6df3147b00

Happy to make a fix so that they line up to default but wanted to check with you first before making a PR. You can see the drift fairly prominently on turbo.build, since I'm using the same logo-title there.

**Comment by fuma-nama** [comment]
Nice, I'm so excited to see it!
Regarding the logo drift, I slightly decreased the font height on docs layout because it was too large when displayed on the sidebar. Perhaps I can take a look later

**Reply by fuma-nama** [reply]
For now, I think I'll keep it like this. It makes sense to me to keep the logo smaller on docs layout (as the logo of fumadocs is just the moon icon)

**Reply by anthonyshew** [reply]
Sorry, I wasn't clear. I was moreso talking about the framework itself rather than `fumadocs.vercel.app`.

For instance, I have what's effectively `pt-4` on the non-docs layouts: 
![CleanShot 2024-06-12 at 08 48 33@2x](https://github.com/fuma-nama/fumadocs/assets/35677084/66cd8917-a793-436b-b3dd-c733588756e8)


Whereas there is `pt-6` here:
![CleanShot 2024-06-12 at 08 49 54@2x](https://github.com/fuma-nama/fumadocs/assets/35677084/c4f2e139-ed37-4d52-8b75-85db254b93be)

I'm using the same logo so I was wondering if they are meant to line up (or if I've messed up somewhere).  I can hack in a fix that makes them line up myself but want to make sure I'm not going to collide with anything you might do for a fix.

**Reply by fuma-nama** [reply]
I actually prefer keeping the padding top here, they are different navigation elements and in a different layout. Leaving a bigger padding top can make the sidebar looks less crowded, you can adjust it with the `bannerProps` on docs layout.

I'm experimenting different styles now, will slightly change the design next release.

**Reply by anthonyshew** [reply]
Might be a bug report but I can see the `bg-green-400` in the resulting HTML here but not the `bg-purple-400`.
```
export default function Layout({
  children,
}: {
  children: ReactNode;
}): JSX.Element {
  return (
    <DocsLayout
      containerProps={{ className: 'bg-green-400' }}
      sidebar={{ bannerProps: { className: 'bg-purple-400' } }}
      tree={repoDocsPageTree}
    >
      {children}
    </DocsLayout>
  );
}

```
![CleanShot 2024-06-13 at 12 33 36](https://github.com/fuma-nama/fumadocs/assets/35677084/a3dc9b77-9a82-4572-ab49-7ab5b1ceebe4)

**Reply by fuma-nama** [reply]
Oops you are right, I fixed the bug, will release a fix soon
